frontend/src/components/LandingPage.tsx

```diff
@@ -17,29 +18,40 @@ const LandingPageContainer = styled.div`
   }
 `;
 
-const Prompt = styled.div`
-  font-size: 20px;
-  font-weight: normal;
-  margin-top: 20px;
-`;
-
 const ImageContainer = styled.div`
   flex: 1;
-  margin-top: 30px;
+  padding-top: 30px;
   width: 100%;
+  background: ${LOGIN_BACKGROUND};
+  display: flex;
+  align-items: center;
 `;
 
 const LandingPageImage = styled.img`
   width: 100%;
+  flex: 1;
 `;
 
 const LandingPageText = styled.div`
   flex: 1;
-
-  @media (min-width: 600px) {
-    margin-left: 1em;
-    margin-top: 60px;
+  margin-left: 1em;
+  margin-top: 60px;
+  margin-right: 1em;
+  @media(min-width: 601px) {
+    margin-top: 120px;
   }
+  
+  
+`;
+
+const Prompt = styled.p`
+ // Not sure why the table works here, but I am using it since otherwise it would take up the full width lol
+  display: table;
```

Comment:
lol i think it's okay. I like to use width: max-content or width: 100%. Either works